src/plugins/inputs/away.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+exports.commands = ["away", "back"];
+
+exports.input = function(network, chan, cmd, args) {
+	if (cmd === "away") {
+		let reason = " ";
+
+		if (args.length > 0) {
+			reason = args.join(" ");
+		}
+
+		network.irc.raw("AWAY", reason);
+
+		return;
+	}
+
+	network.irc.raw("AWAY");
```

Comment:
Aaah, I did not realize that this file was handling both :facepalm:

Reason why I was confused by this: Not long ago, all input files were being called when sending something, one by one, and at that time they were all starting with a check that the command being sent was the one from the file. I forgot we improved that to lookup for the exact command instead.
Prior to that change, I think that line would have been run whatever input was being sent.

So I think we're all good here, let me know if I'm in the wrong again lol.